framework/src/application/schema/application_config_schema.js

```diff
@@ -572,7 +587,11 @@ module.exports = {
 					releaseLimit: 25,
 				},
 				transactions: {
-					maxTransactionsPerQueue: 1000,
+					maxTransactions: 4096,
```

Comment:
It doesn't need the default value, as it is set in the library.
If we put it here, it will be duplicate settings.

Reply:
wouldn't that mean every other config is set to default and the `transactions` needs to be empty?

Reply:
yes, I think all optional thing shouldn't set as `default` in the config, as it is already defined in the library. Although currently it's mixture.
p2p properties and some properties follows this, but not all

Reply:
ok, I will remove here then, and then maybe we can address the p2p config issue in another ticket. but it's good to have consistent approach.

Reply:
👍 but i meant p2p config and some are following this approach, but not others =)